phys2bids/physio_obj.py

```diff
@@ -268,34 +272,34 @@ def check_trigger_amount(self, thr=2.5, num_timepoints_expected=0, tr=0):
         time_offset = self.timeseries[0][timepoints.argmax()]
 
         if num_timepoints_expected:
-            print('Checking number of timepoints')
+            LGR.info('Checking number of timepoints')
             if num_timepoints_found > num_timepoints_expected:
                 timepoints_extra = (num_timepoints_found
                                     - num_timepoints_expected)
-                print(f'Found {timepoints_extra} timepoints'
-                      ' more than expected!\n'
-                      'Assuming extra timepoints are at the end '
-                      '(try again with a more conservative thr)')
+                LGR.info(f'Found {timepoints_extra} timepoints'
+                         ' more than expected!\n'
+                         'Assuming extra timepoints are at the end '
+                         '(try again with a more conservative thr)')
 
             elif num_timepoints_found < num_timepoints_expected:
                 timepoints_missing = (num_timepoints_expected
                                       - num_timepoints_found)
-                print(f'Found {timepoints_missing} timepoints'
-                      ' less than expected!')
+                LGR.warning(f'Found {timepoints_missing} timepoints'
+                            ' less than expected!')
                 if tr:
-                    print('Correcting time offset, assuming missing timepoints'
-                          ' are at the beginning (try again with '
-                          'a more liberal thr')
+                    LGR.warning('Correcting time offset, assuming missing timepoints'
+                                ' are at the beginning (try again with '
+                                'a more liberal thr')
                     time_offset -= (timepoints_missing * tr)
                 else:
-                    print('Can\'t correct time offset, (try again specifying '
-                          'tr or with a more liberal thr')
+                    LGR.error('Can\'t correct time offset, (try again specifying '
+                              'tr or with a more liberal thr')
 
             else:
-                print('Found just the right amount of timepoints!')
+                LGR.info('Found just the right amount of timepoints!')
 
         else:
-            print('Cannot check the number of timepoints')
+            LGR.error('Cannot check the number of timepoints')
```

Comment:
We could change this message to something like:
"you didn't specify the options necessary to perform this action".
If that's a warning, error or info I don't know.